cmd/install_test.go

```diff
@@ -114,7 +116,7 @@ func Test_RewriteKubeconfig(t *testing.T) {
 }
 
 func Test_getHelmURL(t *testing.T) {
-	got := getHelmURL("amd64", "darwin", "v2.14.3")
+	got := helm.GetHelmURL("amd64", "darwin", "v2.14.3")
```

Comment:
@alexellis looks like this may have snuck in during the recent restructure and caused CI failure